.github/workflows/nightly-builds.yml

```diff
@@ -134,6 +145,9 @@ jobs:
             -Dakka.test.timefactor=2 \
             -Dakka.actor.testkit.typed.timefactor=2 \
             -Dakka.test.multi-in-test=false \
+            -Dmultinode.XX:MetaspaceSize=128M \
+            -Dmultinode.Xms256M \
+            -Dmultinode.Xmx256M \
```

Comment:
those are not used now, but if we enable multi-jvm here later it's good to have them in place